core/block_svg.ts

```diff
@@ -1450,17 +1416,6 @@ export class BlockSvg
     }
   }
 
-  /**
-   * Schedule snapping to grid and bumping neighbours to occur after a brief
-   * delay.
-   *
-   * @internal
```

Comment:
Unfortunately we use this in a plugin :/ This also seems like a likely function that people would be using even though it's marked internal. How would you feel about deprecating it instead and removing it at a future date?

Reply:
How do you feel about making `scheduleSnapAndBump` public instead? Since we already established people may want to bump this after programatically moving their blocks.

Reply:
That sounds great!